packages/layout/src/footnotes/mapFootNotesToView.js

```diff
@@ -0,0 +1,59 @@
+
+import createInstance from '../node/createInstance';
+
+/**
+ *  maps footnote elements to a view node 
+ * 
+ * @param {*} footnotes 
+ * @returns a view node
+ */
+function mapFootNotesToView(footnotes) {
+  let processed = [
+    createInstance({
+      type: 'SVG',
+      props: {
+        style: {
+          height: 5,
+        },
+        children: [],
+      },
+    }),
+  ];
+
+  let line = createInstance({
+    type: 'LINE',
+    props: {
+      x1: 0,
+      x2: 1000,
```

Comment:
How is this value being derieved? Does it ensure that we will get horizontal lines across all kinds of page sizes?

Reply:
fixed this by using page width as the width for the line